test/test_helper.rb

```diff
@@ -2,7 +2,7 @@
 require 'this_data'
 
 require 'minitest/autorun'
-require 'mocha/mini_test'
+require 'mocha/minitest'
```

Comment:
Did the name of the component change?

Reply:
Yes!
https://github.com/freerange/mocha/commit/55d66ed4f5e6cad95e1605341ec3c608be2c6833